### PR TITLE
Porting contained sprites and first custom item wave

### PR DESCRIPTION
**Pull request by Alberyk**
Porting the contained sprite var from old code, allowing for less problems with icon storages in the future.

Custom items added:

Antique Pocket Watch - Eric Derringer - xelnagahunter
Mendoza's cap - Chance Mendoza - loow
Ziva's Bandana - Ziva Ta'Kim - sierrakomodo
Fancy Jacket - Zubari Akenzua - filthyfrankster (req_titles: Security Officer, Warden, Head of Security, Detective, Forensic Technician)
Heirloom Unathi Mantle - Sslazhir Yinzr - alberyk
Chich Eyewear - Roxy Wallace - nebulaflare
Data Chip - Roxy Wallace - nebulaflare
Sharpshooter Gloves - Hawk Silverstone - nebulaflare
Data Drive Pendant -  Kyyir'ry'avii Mo'Taki - nebulaflare
Graphite's chassis - Grap..Null - dreamixpl
Skrell Purple Head Cloth - Shkor-Dyet Dom'Pesh - mofo1995
Pocket Altar - Kiara Branwen - nursiekitty
Brown Hat - Avery Bell - serveris6
Pinned Brown Coat - Avery Bell - serveris6

Also, adding old custom synth sprites from old code.
This config should work, if ever needed: https://www.dropbox.com/s/9zi9t9zrcuqfrvg/custom_items.txt?dl=0

### CODE REVIEW

**Comment by LordLag** [comment]
#430 please note as to what is already being added in the future.

**Comment by Alberyk** [comment]
And there we go, should be the same.

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 1]
```diff
@@ -512,6 +516,8 @@ var/global/list/damage_icon_parts = list()
 		wear_id.screen_loc = ui_id	//TODO
 		if(w_uniform && w_uniform:displays_id)
 			overlays_standing[ID_LAYER]	= image("icon" = 'icons/mob/mob.dmi', "icon_state" = "id")
+		else if(wear_id.contained_sprite)
+			overlays_standing[ID_LAYER] = image("icon" = wear_id.icon_state, "icon_state" = "[wear_id.icon_state]_w")
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/update_icons.dm]: `"icon" = wear_id.icon_state` is wrong.
Should be: `"icon" = wear_id.icon`

**Comment by Alberyk** [comment]
Should be fixed, at least the id bit.

**Pull request merged**